.github/workflows/ci.yaml

```diff
@@ -13,40 +13,38 @@ jobs:
   test-unit:
     runs-on: ubuntu-latest
     container:
-      image: golang:1.22.1-bookworm
+      image: golang:1.22.5-bookworm
     steps:
     - name: Checkout code
-      uses: actions/checkout@v3
-    - uses: actions/cache@v3
+      uses: actions/checkout@v4
```

Comment:
Nit: we should probably start pinning actions to SHAs at some point (https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions), and have Dependabot deal with updating them.

A tool that can do this (and more!) for you in seconds is: https://app.stepsecurity.io.

Reply:
iirc, Dependabot is looking after Action versions in Kargo proper. I was surprised at how far behind they were over here. I will be sure to follow up on this.